me.stonegate.tsacdop.json

```
@@ -0,0 +1,65 @@
+{
+    "app-id": "me.stonegate.tsacdop",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "tsacdop",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=me.stonegate.tsacdop"
```

```suggestion
        "--talk-name=org.freedesktop.Notifications"
```

The DBus permission is granted by default with the app-id.